expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -10,13 +10,15 @@
 import java.util.stream.Collectors;
 import no.unit.nva.expansion.model.ExpandedDataEntry;
 import no.unit.nva.expansion.model.ExpandedDoiRequest;
+import no.unit.nva.expansion.model.ExpandedGeneralSupportRequest;
```

Comment:
We will deal with this class later. I have removed most modules in order to reduce the amount of changes per PR. We merge in sandbox branch